.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  - push
+
+jobs:
+  push:
+    name: push
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
```

Comment:
I don't see any use of lfs in the repo?

Reply:
Ah yeah, I originally had the .wasm and other files checked in, but ended up removing them. Don't think it hurts anything to leave this here?

Reply:
yeah doesn't hurt anything, I just saw it and got a little confused. I generally try to avoid LFS unless absolutely necessary since it causes unexpected behavior for people not familiar with it